src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String PATH_TO_INPUT_FILE = "src/main/resources/";
```

Comment:
let's not hardcode paths in reader/writer services, better to pass them as method parameters with the file name itself as a single String

Reply:
ok, fixed )